sourced/ml/cmd/args.py

```diff
@@ -120,7 +120,7 @@ def add_feature_args(my_parser: argparse.ArgumentParser, required=True):
     for ex in extractors.__extractors__.values():
         for opt, val in ex.OPTS.items():
             my_parser.add_argument(
-                "--%s-%s" % (ex.NAME, opt), default=val, type=json.loads,
+                "--{}-{}".format(ex.NAME, opt), default=val, type=json.loads,
```

Comment:
According to our [style guide](https://github.com/src-d/guide/blob/master/engineering/conventions-python.md#code-style) we do not use `format()`.

Reply:
Thanks, reverted!